Update/wata_tips_17.txt

```diff
@@ -134,7 +134,7 @@ void main()
 //　園崎家周辺は監視カメラで守られるほどの厳重ぶりだが、幸い、私は当主跡継ぎの魅音・詩音の姉妹とは面識がある＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　園崎家周辺は監視カメラで守られるほどの厳重ぶりだが、幸い、私は当主跡継ぎの魅音・詩音の姉妹とは面識がある。",
-		   NULL, "The Sonozaki house is strictly guarded by security cameras on the premesis, but thankfully, I've become acquainted with the sisters Mion—the next heir—and Shion.", GetGlobalFlag(GLinemodeSp));
+		   NULL, "The Sonozaki house is strictly guarded by security cameras on the perimeter, but thankfully, I've become acquainted with the sisters Mion—the next heir—and Shion.", GetGlobalFlag(GLinemodeSp));
```

Comment:
revert

Reply:
what's a premesis?

Reply:
I think It's a typo. It's probably supposed to be "premises" (swapped e and i).